packages/modules/src/modules/epics/ContributionsEpicChoiceCards.tsx

```diff
@@ -71,20 +60,31 @@ const contributionType: ContributionType = {
     },
 };
 
+// ContributionsEpicChoiceCards - exported component
+// -------------------------------------------
+export interface ChoiceCardSelection {
+    frequency: ContributionFrequency;
+    amount: number | 'other';
+}
+
+interface EpicChoiceCardProps {
+    selection: ChoiceCardSelection | undefined;
+    setSelectionsCallback: (choiceCardSelection: ChoiceCardSelection) => void;
+    submitComponentEvent?: (event: OphanComponentEvent) => void;
+    currencySymbol: string;
+    amounts: ContributionAmounts;
+}
+
 export const ContributionsEpicChoiceCards: React.FC<EpicChoiceCardProps> = ({
-    amounts,
     selection,
     setSelectionsCallback,
-    countryCode,
     submitComponentEvent,
+    currencySymbol,
+    amounts,
 }: EpicChoiceCardProps) => {
-    const currencySymbol = getLocalCurrencySymbol(countryCode);
-    const countryGroupId = countryCodeToCountryGroupId(countryCode || 'GBPCountries');
-    const countryAmountsData = amounts[countryGroupId];
-
-    // This is the bit we need to change for amounts testing
-    // Ideally we'll fix so the component is pre-supplied with the correct variant
-    const amountsTestVariant = countryAmountsData.control;
+    if (selection == null) {
```

Comment:
It looks like the type of `selection` is `ChoiceCardSelection | undefined`. I wonder if this condition should be tightened up to `selection === undefined` (or maybe just `!selection`). I think the `null` here is a little bit misleading as I don't think `selection` will ever actually be `null` (I think it works because the looser equality check which `==` gives you says that `undefined == null` is true).

Reply:
I just prefer `selection == null` because it captures both `undefined` and `null` states. That way I don't need to care about how others may choose to define and setup the variable being tested for existence. It is the one use of `==` about which I am entirely happy

Reply:
> how others may choose to define and setup the variable
I think Tom's point is that typescript wouldn't allow others to define it as `null`.

The use of typescript (mostly!) means we don't have to worry about defending against unexpected variable types